src/general/MainMenu.cs

```diff
@@ -384,6 +385,37 @@ private void OptionsPressed()
         options.OpenFromMainMenu();
     }
 
+    private void OnDiscordPressed()
+    {
+        // Open the url as a process. The OS will handle selecting the default browser.
+        Process.Start("https://discordapp.com/invite/ECR9E8x");
```

Comment:
These should use the Godot approach to opening an URL. There is at least the translations site open button in the options menu which you can refer to.